dashboard/app/helpers/levels_helper.rb

```diff
@@ -160,6 +160,12 @@ def blockly_options
       level_options['toolbox'] = level_view_options[:toolbox_blocks]
     end
 
+    # NetSim-specific option
+    if @level.game == Game.netsim
```

Comment:
We will eventually have other levels that depend on Pusher.  Is there a way you can have the level tell you whether or not it needs Netsim, rather than having this check at muttiple places in the code?